test/e2e/local-postman.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: executor.kubtest.io/v1
+kind: Executor
+metadata:
+  annotations:
+    meta.helm.sh/release-name: kubtest
+    meta.helm.sh/release-namespace: default
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: local-postman-executor
```

Comment:
why do we need a local-postman-executor?

Reply:
@nicufk To test local postman executor - you can't access one in Kubernetes (without messing with some networking stuff) 
- notice that it's attached to `local-postman/collection` type - you can manage as many postman executors as you want.